### ISSUE TITLE
Add OSF as MDFP

### ISSUE THREAD

**Issue by saradbowman** [EFForg/privacybadger#2085]
Hi there, 

We've had reports of Privacy Badger blocking access to some of our sites. We build open source web apps for research management; the core of these is the OSF (osf.io). The OSF API powers a variety of services. Some of these are our preprint server, OSF Preprints (osf.io/preprints), and various community-lead preprint servers under their own domains (agrixiv.org, psyarxiv.org, etc.). As a result, Privacy Badger recognizes api.osf.io as a third-party tracker. 

Can you add the OSF, its API, and its preprint services to the MDFP list? They are:

api.osf.io
osf.io
agrixiv.org
arabixiv.org
eartharxiv.org
ecsarxiv.org
engrxiv.org
frenxiv.org
marxiv.org
mindrxiv.org
paleorxiv.org
psyarxiv.com
thesiscommons.org

Many thanks

**Comment by ghostwords** [comment]
Hello! Yep, `api.osf.io` is indeed unnecessary, because our list of first party domains only supports base (eTLD+1) domains (is this documented anywhere? we should document it).

**Comment by ghostwords** [comment]
Do you know how to reproduce Privacy Badger learning to block `api.osf.io`? Does this involve logging in?

**Comment by saradbowman** [comment]
That's a good question - I was logged in when I triggered it. Is there a way for me to un-teach Privacy Badger about api.osf.io and try again (logged out)?

**Comment by ghostwords** [comment]
Yep, you can make your Badger forget domains by visiting the options page, selecting the Tracking Domains tab, searching for the domain(s), and clicking the "x" next to each domain you want to remove.

**Comment by saradbowman** [comment]
Thanks! I tested again; I did not need to be logged in for Privacy Badger to block api.osf.io

**Comment by ghostwords** [comment]
I see what's going on. Visiting osf.io directly sets Google Analytics cookies (#367). Visiting any site that uses the OSF API then sends those cookies to `api.osf.io`, which Privacy Badger flags as tracking.

**Comment by ghostwords** [comment]
Would OSF be able and willing to abide by the [EFF Do Not Track policy](https://www.eff.org/dnt-policy)'s requirements on `api.osf.io`? If so, posting the policy there will tell Privacy Badger to always allow loading of resources from `api.osf.io`.

**Comment by ghostwords** [comment]
Added note regarding MDFP and base domains in 246d49ed0b917eb2a99e4d42468f073cf00ecfa1.
